docs/RELEASE.md

```diff
@@ -170,5 +148,7 @@ Describes how developers should merge bug and security fixes.
 3. Audit happens with priority.
 4. It is merged into `master`.
 5. Dev adds the `A4-needs-backport` label.
-6. It is automatically back-ported to `stable`.
-7. The fix will be released in the next *Stable* release. In urgent cases, a release can happen earlier.
+6. It is automatically back-ported to `stable` and merged by a release engineer.
+7. The fix will be released in the next *Stable patch* release. In urgent cases, a release can happen earlier.
+
+The release itself is similar to [the new stable release](#steps-to-execute-a-new-stable-release) process without the branching-off step, as the branch already exists and depending on the patch (whether it is for the current `stable` release or one of the previous ones) the binary build can be skipped and only crates and GitHub publishing is done.
```

Comment:
dumb question: why do we skip binaries release for bug/security fixes?

Reply:
There was a discussion about it some time ago in the release coordination chat and on the forum: https://forum.parity.io/t/polkadot-binary-releases/2429/5
If there is something critical needed to be fixed, we will always build and attach those to the release.